dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/view/EventChangeLog.java

```diff
@@ -37,10 +37,16 @@ public record EventChangeLog(
     @JsonProperty String type,
     @JsonProperty Change change) {
 
-  public record Change(@JsonProperty DataValueChange dataValue) {}
+  public record Change(
+      @JsonProperty DataValueChange dataValue, @JsonProperty EventPropertyChange eventProperty) {}
 
   public record DataValueChange(
       @JsonProperty UID dataElement,
       @JsonProperty String previousValue,
       @JsonProperty String currentValue) {}
+
+  public record EventPropertyChange(
```

Comment:
Is the `Event` prefix needed? We know we are dealing with event changes from the context given by the url path/type/package.

We removed many `Tracker` prefixed from our code as that was not giving any extra information and only taking up space. This prefix feels the same to me unless I am missing something.